### PR TITLE
Frontend/landlord dashboard

### PR DESCRIPTION
**Pull request by garrickhenne**

### CODE REVIEW

File: frontend/src/helpers/LandlordHelpers.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export const calculateAverageRatings = (reviews) => {
+  const repairsAverage = reviews.reduce((prev, curr) => prev + curr.repair, 0) / reviews.length;
+  const respectAverage = reviews.reduce((prev, curr) => prev + curr.respect, 0) / reviews.length;
+  const safetyAverage = reviews.reduce((prev, curr) => prev + curr.healthSafety, 0) / reviews.length;
+  const overall = (repairsAverage + respectAverage + safetyAverage) / 3;
```
Comment (robertshum) [inline frontend/src/helpers/LandlordHelpers.js:5]: I think overallScore is already calculated per review:

```js
  const review: HydratedDocument<IReview> = new Review({
    title: title,
    description: desc,
    sentiment: sentiment,
    healthSafety: healthSafety,
    respect: respect,
    repair: repair,
    overallScore: calculatedScore,
    userId: userId,
    landlordId: landlordId,
  });
  ```
  
  so you could theoretically just use reduce again :)

**Review by robertshum — COMMENTED** [review]
Other than the overallScore formula, looks good!  Gonna be a sick dashboard.

...I'm gonna pretend I understand the math that's going on :O

File: frontend/src/helpers/LandlordHelpers.js (hunk 1 above)
Comment (garrickhenne) [reply frontend/src/helpers/LandlordHelpers.js:5]: Do we have a place in backend to update overall score? If not i can add it to this PR

Comment (robertshum) [reply frontend/src/helpers/LandlordHelpers.js:5]: only when you submit an edited review, it will recalculate the new overall score

Comment (robertshum) [reply frontend/src/helpers/LandlordHelpers.js:5]: it's found in `ReviewService.ts` `getCalculatedReviewScore`

File: frontend/src/components/Review/Score.jsx
Comment (robertshum) [inline frontend/src/components/Review/Score.jsx:1]: Thanks for changing the class->className totally forgot about the warnings

**Review by robertshum — APPROVED** [review]
Looks good to me!

**Pull request merged**